libraries/AP_PiccoloCAN/AP_PiccoloCAN.h

```diff
@@ -15,8 +15,7 @@
  * Author: Oliver Walters
  */
 
-#ifndef AP_PICCOLOCAN_H_
-#define AP_PICCOLOCAN_H_
+#pragma ONCE
```

Comment:
i haven't seen ONCE in upper case used before, better use lower case in case it matters for some compiler

Reply:
Good point - fixed in https://github.com/ArduPilot/ardupilot/pull/12947/commits/f626394871907311cb609f741f71c085588c212c